examples/js/hello-sdl/readme.md

```diff
@@ -1,8 +1,8 @@
 ## SDL JavaScript App Startup Instructions
 1) Place the vanilla JS SDL.min.js file into this directory
-1) Request a [Manticore instance](https://smartdevicelink.com/resources/manticore/)
-1) Open two terminal sessions and `cd` both of them into `./examples/js/hello-sdl`
-1) In one terminal session, run `java -jar proxy.jar m.sdl.tools [PORT]`, where `[PORT]` is the one given to you by Manticore
-1) In the other terminal session, run `npm install` then `npm start`.
-1) Your browser should automatically open and the test app should appear on your Manticore UI.
-1) Click on the app in Manticore and observe the series of `Show` RPCs performed. Logs can be seen in your browser's JavaScript console.
\ No newline at end of file
+2) Request a [Manticore instance](https://smartdevicelink.com/resources/manticore/)
+3) Open the file `./examples/js/hello-sdl/index.html` in any text editor and find setTransportConfig. setTransportConfig uses the old port number 5050 change it to 2020.
```

Comment:
The instructions say to use Manticore, so the port wouldn't be 2020, it will be port 444. The transport config would also need to change its URL to what Manticore's "WS URL" says.